examples/hitl/main.py

```diff
@@ -73,18 +70,15 @@ def start_optimization(
     while True:
         running_trials = study.get_trials(deepcopy=False, states=(TrialState.RUNNING,))
         if len(running_trials) >= n_batch:
+            time.sleep(1)  # Avoid busy-loop
             continue
         suggest_and_generate_image(study, artifact_backend)
 
 
 def main() -> NoReturn:
     tmp_path = os.path.join(os.path.dirname(__file__), "tmp")
 
-    # 1. Create RDBStorage
-    url = "sqlite:///db.sqlite3"
-    storage = optuna.storages.RDBStorage(url=url)
-
-    # 2. Create Artifact Storage
+    # 1. Create Artifact Store
```

Comment:
[question] Why did we move those line into `start_optimization`?

Reply:
To just simplify the code. Thank you for your question.